src/Paysera/Bundle/JavascriptGeneratorBundle/Service/Generator/EntityGenerator.php

```diff
@@ -43,13 +43,10 @@ public function generate(ApiDefinition $definition) : array
                 ]
             );
 
-            $item = new SourceCode();
-            $item
+            $items[] = (new SourceCode())
                 ->setFilepath(sprintf('%s/entity/%s.js', $this->sourceDir, $type->getName()))
```

Comment:
why here (and in `DateFactoryGenerator.php`) you prepend `sourceDir` and in `GitIgnoreGenerator.php` - do not?